test/e2e/soak_namespace.go

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
For consistency, I'd prefer this to be in test/e2e/namespace.go

$ ls -1 test/e2e
cadvisor.go
certs.go
density.go
docker_containers.go
downward_api.go
driver.go
empty_dir.go
es_cluster_logging.go
events.go
kubectl.go
kubelet.log
monitoring.go
networking.go
pd.go
pods.go
rc.go
scale.go
secrets.go
service.go
shell.go
ssh.go
util.go